### PR TITLE
Added more expressive module docstring to batch.py

### PR DESCRIPTION
**Pull request by jgeewax**

### CODE REVIEW

**Comment by jgeewax** [comment]
Looks like lint is failing... but not saying why. Running pylint says the line is too long... Any ideas how I could make the line shorter without cutting the hyperlink? @tseaver

**Comment by tseaver** [comment]
@jgeewax Pylint only allows hyperlinks > 80 characters if they are standalone on the line.  E.g., https://github.com/GoogleCloudPlatform/gcloud-python/blob/6283ab9c9d7fa06f6800079bd882b573c4de7cf7/gcloud/pubsub/api.py#L28

**Comment by dhermes** [comment]
Also the link has 84 chars, 8 of which are `https://`, so you could just use the link on its own line

**Comment by jgeewax** [comment]
Assigning to @tseaver for review

File: gcloud/datastore/batch.py [hunk 1]
```diff
@@ -12,7 +12,15 @@
...
 
-"""Create / interact with a batch of updates / deletes."""
+"""Create / interact with a batch of updates / deletes.
+
+Batches provide the ability to execute multiple operations
+in a single request to the Cloud Datastore API.
+
+See
+https://cloud.google.com/datastore/docs/concepts/entities#Datastore_Batch_operations
+for more information.
```
Comment (tseaver) [inline gcloud/datastore/batch.py]: I think I'd leave off the `for more information`.

Comment (jgeewax) [inline gcloud/datastore/batch.py]: Done.

**Comment by tseaver** [comment]
LGTM

**Pull request merged**